Vulkan-Pipeline-Assistant/PipelineConfig.h

```diff
@@ -74,6 +74,7 @@ namespace vpa {
         VkBool32 logicOpEnable = VK_FALSE;
         VkLogicOp logicOp = VK_LOGIC_OP_COPY;
         float blendConstants[4] = { 0.0f, 0.0f, 0.0f, 0.0f };
+
```

Comment:
Avoid adding whitespace unnecessarily

Reply:
Removed this line and a couple of other blank lines in latest push